test/test_tree_2d_advection.jl

```diff
@@ -13,7 +13,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_basic.jl"),
       # Expected errors are exactly the same as in the parallel test!
       l2   = [8.311947673061856e-6],
-      linf = [6.627000273229378e-5])
+      linf = [6.627000273229378e-5],
+      coverage_override = (maxiters=10^5,))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
I chose to run at least one test (the simplest one) for the full time span.